### PR TITLE
telemetry integration

### PR DESCRIPTION
**Pull request by vijayfractl**

### CODE REVIEW

File: src/agentlang/telemetry.cljc [hunk 1]
```diff
@@ -0,0 +1,55 @@
+(ns agentlang.telemetry
+  (:require [agentlang.util :as u]
+            [agentlang.util.logger :as log]
+            [agentlang.util.http :as http]
+            [agentlang.lang.datetime :as dt]
+            [agentlang.datafmt.json :as json]
+            [agentlang.component :as cn]
+            [agentlang.global-state :as gs]))
```
Comment (pratik-fractl) [inline src/agentlang/telemetry.cljc:8]: I don't think the telemetry support will have any cljs component, and looks like it's only clj as of now. Maybe it's better to keep it only as clj?

Comment (vijayfractl) [reply src/agentlang/telemetry.cljc:8]: I kept it as cljc because the call-sites don't have to bother with wrapping telemetry call in `#?(:clj ...)`

**Review by pratik-fractl — DISMISSED** [review]

File: src/agentlang/telemetry.cljc (hunk 1 above)
Comment (pratik-fractl) [reply src/agentlang/telemetry.cljc:8]: Okay. Makes sense.

**Comment by vijayfractl** [comment]
Example config:

```clojure
{:service {:port 8000}
 :telemetry {:host "https://telemetry-service-dev.fractl.io"}
 :connection-manager
 {:username #$ CM_USERNAME
  :password #$ CM_PASSWORD}}
```

**Review by kumarshantanu — APPROVED** [review]

**Review by kumarshantanu — APPROVED** [review]

**Pull request merged**